repos/spack_repo/builtin/packages/py_zfit/package.py

```diff
@@ -61,12 +65,12 @@ class PyZfit(PythonPackage):
         depends_on("py-tensorflow@2.16.2:2.19", when="@0.25.0:")
         depends_on("py-tensorflow-probability@0.25:0.26", when="@0.25.0:")
 
-        depends_on("py-tensorflow@2.16.2:2.18", when="@0.24.3:")
+        depends_on("py-tensorflow@2.16.2:2.18", when="@0.24.3:0.24")
         depends_on("py-tensorflow@2.18", when="@0.24:0.24.2")
-        depends_on("py-tensorflow-probability@0.25", when="@0.24:")
+        depends_on("py-tensorflow-probability@0.25", when="@0.24:0.24")
```

Comment:
FYI. You could just use `"@0.24"` (see https://spack.readthedocs.io/en/latest/packaging_guide_creation.html#specifying-versions-in-other-directives).

Reply:
makes sense, thanks!